website/src/app/app.component.html

```diff
@@ -8,8 +8,8 @@
             </button>
             <span class="page-header">UQ Secat's</span>
             <div fxFlex="grow"></div>
-            <span><button type="button" md-button><img
-                    src="../assets/images/github-circle-white-transparent.svg" alt=""> GITHUB</button></span>
+            <span><a href="https://wisebaldone.github.io/uq-secat/"><button type="button" md-button><img
```

Comment:
Instead of wrapping a button in a anchor tag you can do the following:

`<a href="https://github.com/wisebaldone/uq-secat/" type="button" md-button><img
                    src="../assets/images/github-circle-white-transparent.svg" alt=""> GITHUB</a>`

By replacing the button tags with anchor tags but leaving the md-button and type button it renders it as if it was the original button which fancy effects and all.